src/xtgeo/surface/_regsurf_roxapi.py

```diff
@@ -134,7 +134,15 @@ def export_horizon_roxapi(
     _check_stypes_names_category(roxutils, stype, name, category)
 
     logger.info("Surface from xtgeo to roxapi...")
-    _roxapi_export_surface(self, roxutils.project, name, category, stype, realisation)
+    use_srf = self
+    if self.yflip == -1:
+        # roxar API cannot handle negative increments
```

Comment:
Should this be more generalized to more than the `-1` case then?

Reply:
It is either 1 or -1